addOns/soap/src/test/java/org/zaproxy/zap/extension/soap/WSDLCustomParserTestCase.java

```diff
@@ -92,8 +112,10 @@ void createSoapRequestTest() {
         assertNull(result);
     }
 
-    @Test
-    void addParameterShouldUseValueGeneratorWhenAvailable() {
+    @ParameterizedTest
+    @EmptySource
+    @ValueSource(strings = {"generated"})
```

Comment:
Add an empty string?

Reply:
[EmptySource](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/params/provider/EmptySource.html)

Reply:
Oh, my bad, I missed that.